src/server/scripts/Northrend/Naxxramas/boss_patchwerk.cpp

```diff
@@ -21,20 +21,20 @@ enum Spells
     SPELL_HATEFUL_STRIKE_25         = 59192,
     SPELL_FRENZY                    = 28131,
     SPELL_BERSERK                   = 26662,
-    SPELL_SLIME_BOLT                = 32309,
+    SPELL_SLIME_BOLT                = 32309
 };
 
 enum Events
 {
     EVENT_HEALTH_CHECK              = 1,
-    EVENT_SPELL_HATEFUL_STRIKE      = 2,
-    EVENT_SPELL_SLIME_BOLT          = 3,
-    EVENT_SPELL_BERSERK             = 4,
+    EVENT_HATEFUL_STRIKE            = 2,
+    EVENT_SLIME_BOLT                = 3,
+    EVENT_BERSERK                   = 4
 };
 
 enum Misc
 {
-    ACHIEV_TIMED_START_EVENT        = 10286,
+    ACHIEV_TIMED_START_EVENT        = 10286
```

Comment:
why? it's ok to keep the comma always, so if you add a new row you don't need to change the existing ones ;)

Reply:
I would go further ahead and remove the enum completely since an abstracted user-defined type called `Misc` is so unnecessary. This should instead be a `const uint32 `. Assuming the script for Patchwerk is perfect this enum is not needed.